### PR TITLE
Scma/spring cloud

### PR DESCRIPTION
**Pull request by RoadRunner14**
Please check the version with Spring Cloud

### CODE REVIEW

File: arena/pom.xml [hunk 1]
```diff
@@ -53,32 +58,32 @@
...
-                        <phase>package</phase>
-                        <goals>
-                            <goal>single</goal>
-                        </goals>
-                    </execution>
-                </executions>
-            </plugin>
-        </plugins>
-    </build>
+<!--    <build>-->
```
Comment (martyschaer) [inline arena/pom.xml]: Commented out code should be deleted if it's no longer in use.

File: arena/src/main/java/ch/bfh/zodnrk/arena/model/Hero.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
+package ch.bfh.zodnrk.arena.model;
+
+import org.springframework.hateoas.RepresentationModel;
+
+public class Hero extends RepresentationModel<Hero> {
```
Comment (martyschaer) [inline arena/src/main/java/ch/bfh/zodnrk/arena/model/Hero.java:5]: Why do we need to duplicate the Hero and Party class in the Arena Module?

File: pom.xml [hunk 3]
```diff
@@ -17,7 +17,7 @@
 
     <properties>
         <java.version>11</java.version>
-        <spring-cloud.version>Hoxton.SR9</spring-cloud.version>
+        <spring-cloud.version>Hoxton.SR7</spring-cloud.version>
```
Comment (cyrillbolliger) [inline pom.xml:20]: Why the downgrade? Feign?

File: arena/src/main/java/ch/bfh/zodnrk/arena/model/Hero.java (hunk 2 above)
Comment (RoadRunner14) [reply arena/src/main/java/ch/bfh/zodnrk/arena/model/Hero.java:5]: Hero and Party classes are in all module part of it.

File: pom.xml (hunk 3 above)
Comment (RoadRunner14) [reply pom.xml:20]: This was an advice of Roger

**Review by RoadRunner14 — COMMENTED** [review]
Please have a look on correction

**Comment by cyrillbolliger** [comment]
> Please have a look on correction

Thnx. Looks great to me.

**Comment by martyschaer** [comment]
LGTM

**Review by martyschaer — APPROVED** [review]

**Pull request merged**